corehq/blobs/tests/test_atomic.py

```diff
@@ -60,6 +63,14 @@ def test_delete_outside_context(self):
         with self.assertRaises(InvalidContext):
             db.delete(BytesIO(b"content"))
 
+    def test_expire(self):
+        meta = self.db.put(BytesIO(b"content"), meta=new_meta())
+        self.assertIsNone(meta.expires_on)
+        with AtomicBlobs(self.db) as db:
+            db.expire(meta.parent_id, key=meta.key)
+        meta = db.metadb.get(parent_id=meta.parent_id, key=meta.key)
+        self.assertGreater(meta.expires_on, datetime.utcnow())
```

Comment:
nit: might want to consider writing two tests
1. check default timing of 120 mins
2. set a value and check that that is set correctly

Reply:
> One must draw a line at some point when adding tests. 

I don't agree with that. It's about the cost of issues being caused compared to advantage one has by not writing all tests and i am not sure if its possible to evaluate that correctly. I like it when tests at least mirror the code base i.e at least test exactly what the code is doing and plus the edge cases or extra goodies. Makes it easier to understand and read tests after seeing the code if one is making changes. 
May be this particular test is not that critical but then that again would change from dev to dev.
Also, Not asking you to change this test.